charts/studio/templates/ci-secret-docker-registry.yaml

```diff
@@ -0,0 +1,15 @@
+{{/*
+  The purpose of this file is to create dockerconfigjson secret o pull images during the chart-testing
+  tool execution in the CI. This is needed because the chart-testing tool does not support the use of
+  private registries. This file is only used in the CI, and it is not used in the deployment of the
+  chart.
+*/}}
+{{- if (hasKey .Values "ci") }}
+apiVersion: v1
+data:
+  .dockerconfigjson: {{ include "dockerconfig" . | b64enc | quote }}
```

Comment:
It should be renamed too
 
https://github.com/iterative/helm-charts/pull/113/files#diff-1e4f080b0de18f77d739f894999ddc1474b09c387463f24cc2de4e774799d523R133